src/components/structures/MatrixChat.tsx

```diff
@@ -1132,6 +1133,29 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 ));
             }
         }
+
+        const client = MatrixClientPeg.get();
+        const plEvent = roomToLeave.currentState.getStateEvents(EventType.RoomPowerLevels, '');
+        const plContent = plEvent ? (plEvent.getContent() || {}) : {};
```

Comment:
```suggestion
        const plContent = plEvent ? plEvent.getContent() : {};
```

`getContent` already does the `|| {}` step